### ISSUE TITLE
MS Teams: NVDA is narrating the "<username> is out of office and may not respond" notification information twice when user presses Enter on the group message chat.

### ISSUE THREAD

**Issue by HarishTerala** [nvaccess/nvda#8950]
### Steps to reproduce:
1. Open NVDA and launch MS Teams app.
2. Navigate to the "Chat" Appbar element and press Enter to activate it.
3. Navigate to any group message chat and press Enter.
4. The focus will land on the "Type a new message" edit box.
5. Observe the narration of NVDA.

### Actual behavior:
When user presses Enter on the group message chat and focus lands on the "Type a new message" edit box, NVDA is narrating the "<username> is out of office and may not respond" notification information twice.

### Expected behavior:
NVDA should narrate the "<username> is out of office and may not respond" notification information only once.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
2018.2.3

#### Windows version:
Win10 1809

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
I cannot reproduce this in MS Teams 1.6 and NVDA 2022.4. @HarishTerala are you still having this issue with MS Teams 1.6?

**Comment by Adriani90** [comment]
We don't have any updates from the initial author upon request  since many years. I am closing as abandoned. If you are still having this issue, please comment and we can reopen.
